kong/plugins/ai-prompt-decorator/schema.lua

```diff
@@ -0,0 +1,80 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+-- local prompt_schema = {
+--   type = "record",
+--   required = false,
+--   fields = {
+--     { role = { type = "string", required = false, one_of = { "system", "assistant", "user" }, default = "system" }},
+--     { content = { type = "string", required = true } },
+--     { position = { type = "string", required = true, one_of = { "BEGINNING", "AFTER_FINAL_SYSTEM", "AFTER_FINAL_ASSISTANT" "END" }, default = "BEGINNING" }},
+--   }
+-- }
+
+local prompt_record = {
+  type = "record",
+  required = false,
+  fields = {
+    { role = { type = "string", required = true, one_of = { "system", "assistant", "user" }, default = "system" }},
+    { content = { type = "string", required = true } },
+  }
+}
+
+local prompts_record = {
+  type = "record",
+  required = false,
+  fields = {
+    { prepend = {
+      type = "array",
+      description = [[Insert chat messages at the beginning of the chat message array.
+                      This array preserves exact order when adding messages.]],
```

Comment:
This description string is going to have a newline and a bunch of whitespace in between the first and second sentence. Is that the intent? If not, string concatenation is the better way to construct this string:


```suggestion
      description = "Insert chat messages at the beginning of the chat message array. "
                 .. "This array preserves exact order when adding messages.",
```